src/pykx/pandas_api/pandas_meta.py

```diff
@@ -228,6 +228,21 @@ def max(self, axis=0, skipna=True, numeric_only=False):
             res
         ), cols)
 
+    @convert_result
+    def idxmax(self, axis=0, skipna=True, numeric_only=False):
+        res, cols = preparse_computations(self, axis, skipna, numeric_only)
+        maximums = self.max(axis, skipna, numeric_only)
+        num_str = '9h$' if numeric_only else ''
+        func_str = '(::)' if axis==0 else 'column_names'
+        if(numeric_only):
+            (_, column_names) = _get_numeric_only_subtable_with_bools(self)
+        else:
+            column_names = q('cols', self)
+        op= q("{[row;col;maximums;column_names]"
+              f"{func_str}[({num_str}row) ?' maximums[col]]"
+              "}")
```

Comment:
Minor: Try not to use double quotes `"` if possible. You would still need them for line 242, but it's not needed elsewhere on this string.

Also minor: there is a space missing on the assign `op = `